src/test/java/uk/gov/hmcts/reform/coh/service/QuestionServiceTest.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.coh.service;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.coh.Notification.QuestionNotification;
+import uk.gov.hmcts.reform.coh.domain.OnlineHearing;
+import uk.gov.hmcts.reform.coh.domain.Question;
+import uk.gov.hmcts.reform.coh.domain.QuestionState;
+import uk.gov.hmcts.reform.coh.repository.QuestionRepository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+@RunWith(SpringRunner.class)
+public class QuestionServiceTest {
+
+    @Mock
+    private QuestionRepository questionRepository;
+
+    @Mock
+    private QuestionStateService questionStateService;
+
+    private QuestionState drafted = new QuestionState("DRAFTED");
+    private QuestionState issued = new QuestionState("ISSUED");
+    private Question question;
+
+    private static final Long ONE = 1L;
+
+    @Mock
+    private QuestionNotification questionNotification;
+
+    @Mock
+    private OnlineHearingService onlineHearingService;
+
+    private Long questionId = Long.valueOf(2000);
+
+    @InjectMocks
+    private QuestionService questionService;
+
+    @Before
+    public void setup(){
+        question = new Question();
+        question.setQuestionState(issued);
+
+        when(questionRepository.findById(any(Long.class))).thenReturn(Optional.ofNullable(question));
+        when(questionRepository.save(any(Question.class))).thenReturn(new Question());
+
+        when(questionNotification.notifyQuestionState(any(Question.class)))
+                .thenReturn(true);
+
+        when(questionStateService.retrieveQuestionStateById(anyInt())).thenReturn(drafted);
+        when(onlineHearingService.retrieveOnlineHearing(any(OnlineHearing.class))).thenReturn(Optional.of(new OnlineHearing()));
+    }
+
+    @Test
+    public void testCreateQuestion() {
+        when(questionRepository.save(question)).thenReturn(question);
+        when(questionStateService.retrieveQuestionStateById(1)).thenReturn(drafted);
+
+        /**
+         * This needs to be fixed so that online hearing id is an attribute of question
+         */
+        Question newQuestion = questionService.createQuestion(question, UUID.randomUUID());
```

Comment:
See previous comment about passing in the online hearing as part of the question